src/main/java/com/solvd/pages/common/LoginPageBase.java

```diff
@@ -0,0 +1,29 @@
+package com.solvd.pages.common;
+
+import org.openqa.selenium.WebDriver;
+
+public abstract class LoginPageBase extends PageBase {
+    public LoginPageBase(WebDriver driver) {
```

Comment:
it's a good practice to leave a blank line between the class declaration and the first field